packages/medulas-react-components/src/context/ToastProvider/Toast/index.tsx

```diff
@@ -29,14 +29,13 @@ export const Toast = ({
 }: Props): JSX.Element => {
   return (
     <Snackbar
+      id="toast-provider"
```

Comment:
I was required to add this id to check for the Toast in DOM tests.

Reply:
SGTM.